templates/vvt/__form.html.twig

```diff
@@ -132,6 +132,7 @@
         <div class="mt-4 row">
             <div class="col-md-3">
                 {{ form_label(form.kategorien) }}
+                {{ form_help(form.kategorien) }}
```

Comment:
Ich fände es ziemlich praktisch wenn hier die Kategorie mit einem Link angezeigt wird, dann kann man von da aus direkt zu der Kategorie springen, denn jetzt muss man erst zu der Kategorie Übersicht, und dann zu der Entsprechenden Kategorie weiter navigieren. Das selbe dann in der Kategorie, Dort einfach pro Löschkonzept ein Link zu dem Entsprechenden Löschkonzept.